### DISCUSSION TITLE
discussion: sync with the apache Iceberg community

### DISCUSSION THREAD

**Discussion by neverchanje** [icelake-io/icelake#86] [General]
Hi there,

There is a thread regarding the C++/Rust support for Iceberg in the community's dev mail list. Here is the link: https://lists.apache.org/thread/7njzq6b0m9qbjmbtgrtkhmj2nnmhso4t I wonder if you fellows can reply to this thread and ask for the feedbacks from the Iceberg community? I think most community members would welcome this project. However, there will be a lot to discuss in terms of the technical design, in which you may need the professional advice from the community. With the advice and feedbacks, we will be able to see if this project is heading towards the right direction.

**Comment by Xuanwo** [comment]
Great idea! Of course, we need to synchronize with the Apache Iceberg community. However, asking for feedback at this stage may not be a good idea. We need to shape our project and make it work first.

**Comment by neverchanje** [comment]
I agree not asking for feedbacks when it's immature. But I suggest that you may need to first let the community be aware of this project and make sure there is no key members who downvote for it. In addition, there may also be some members who are interested in the early contribution.

Ultimately, I personally suggest that this project would be better to donate back to the community in order to gain the official maintenance and acquire a larger contributor group. Therefore, it'd be better to let the community engage as early as possible.

**Comment by liurenjie1024** [comment]
> I agree not asking for feedbacks when it's immature. But I suggest that you may need to first let the community be aware of this project and make sure there is no key members who downvote for it. In addition, there may also be some members who are interested in the early contribution.
> 
> Ultimately, I personally suggest that this project would be better to donate back to the community in order to gain the official maintenance and acquire a larger contributor group. Therefore, it'd be better to let the community engage as early as possible.

Good suggestion. I'm fine with letting the community know the things we are working on.
